src/lib/components/CopyToClipboard/CopyToClipboard.svelte

```diff
@@ -0,0 +1,74 @@
+<script lang="ts">
```

Comment:
I figured it made sense to break this out into a separate component since it could be useful elsewhere. I wasn't sure if it needed its own story though, so I only added tests (it can be seen in the `ContactInfoBox` story though).